### ISSUE TITLE
mobile os

### ISSUE THREAD

**Issue by sasha240100** [runtimejs/helloworld#2]
Thank you for your work! :)
I have some questions:
- can I build is for my smartphone with runtime.js ?
- how can I detect touch events?

**Comment by iefserge** [comment]
@sasha240100 nope, currently there is no support for ARM CPUs, it wouldn't work on the smartphone.

**Comment by sasha240100** [comment]
@iefserge, will it work in future?

**Comment by sasha240100** [comment]
@iefserge, hello from Ukraine:D

**Comment by iefserge** [comment]
@sasha240100 hey! :)

> will it work in future?

not sure, that requires a lot of work to support it. What's the use of such a runtime.js powered mobile device?

**Comment by sasha240100** [comment]
@iefsege, it would be interesting to create operation system for mobile)). But the problem is that JavaScript is the only language I know :c
